src/components/footer/index.tsx

```diff
@@ -18,7 +19,10 @@ function Footer({
 }: FooterProps): React.ReactElement {
   return (
     <footer className={classNames(s.root, className)}>
-      <InlineSvg className={s.logo} src={svgHashicorpLogo} />
+      <a href="https://www.hashicorp.com/">
+        <InlineSvg className={s.logo} src={svgHashicorpLogo} />
+        <VisuallyHidden>HashiCorp home page</VisuallyHidden>
+      </a>
```

Comment:
I would opt for using `aria-label` here on the anchor element instead of visually hidden text, is there a particular reason you opted for the latter? We get a lot of things `VisuallyHidden` does for free by using `aria-label`.
- Secondary comment, I'd also add a little bit to the front of the text you have here -- "_Go to_ HashiCorp home page" -- because the additional "go to" text that expresses exactly what will happen when the presented element is activated.

---

💭 Ideally, we'd be able to only add an `alt` prop to `InlineSvg` (no `aria-label`), but that doesn't behave as I immediately expected it to. It applies the `alt` prop to the containing `div` and I'm not sure exactly how we'd pass that prop down to the `svg`. There's two things that providing an `alt` prop would do for us here on an a11y front:

- give an alt text on the image (svg specifically in this case)
- give discernible text to the link

Reply:
Sweet, will go with `aria-label` - greatly appreciate the input here 🙇‍♂️

Re: passing `alt` to the SVG, this is always something that makes my head spin a bit (eg when thinking about setting up a repo for logos, & whether it makes sense to include `title` in logo SVGs). It seems like an accurate description of the logo SVG (`HashiCorp logo` or something) would be in tension with an accurate description of the link (I think `Go to HashiCorp home page` is great as you suggested). To clarify, if we did have the `alt` prop working the way we want, would we do `alt="Go to HashiCorp home page"`? Or `alt="HashiCorp logo"`? Or something else entirely 🤔

Reply:
You're on exactly the right track! An accurate description of the logo SVG _would_ be in tension with an accurate description oof the link. This is a good example of where we would not want to "automate" `alt` or `title` in a logos repo, unless we wanted to set up the repo to expose two kinds of components:

- A plain svg/img component where the `alt` is generated to be  `"<Product name> logo"`
- A svg/img component within a link that could maybe look something like this in order to generate a useful alt text:
  ```jsx
  const SvgLogoLink = ({ href, product, pageName ...stuff }) => {
    return (
      <a href={href}>
        {/* src would be based off of the product prop */}
        <svg alt={`Go to ${product} ${pageName}`} {...stuff} />
      </a>
    )
  }
  ```

My default stance is not jumping to automate things like this too quickly, because both the `alt` text and component API deserve great care. Your head-spinning feeling is totally valid! It's a complex issue that takes some time to get used to.

Reply:
I've added docs in our Accessibility Notion page! Link here for posterity. 😊 

https://www.notion.so/hashicorp/Accessibility-4c9cc7245b09498cbedee2e1dde99727#3e0cc537c160489c97aa1c4d05b4764f